docs/user-guide/concepts.md

```diff
@@ -218,6 +221,25 @@ probably not a great idea.
 You can create a subnamespace from the command line via `kubectl hns create child
 -n parent`.
 
+<a name="hierarchical-resource-quota">
+
+### Hierarchical Resource Quotas
+
+When you want to give some amount of resources to `team-a`, and want them to be able
+to distrtibute their resources between themselves, you create a HierarchicalResourceQuota
+in namespace `team-a`. any member from `team-a` can create his own HierarchicalResourceQuota
+in his subnamespace, and the sum of all reasources from all the subnamespaces of the
+members wont be over the amount of resources that is configured in
+HierarchicalResourceQuota of namespace `team-a`.
+
+when creating HierarchicalResourceQuota, HNC automatically creates ResourceQuota
```

Comment:
Capitalize: when -> When

But maybe I'd replace this whole paragraph with this:

```
To implement hierarchical quotas, HNC automatically creates `ResourceQuota` objects in each
affected namespace. These objects should not be modified or relied on by users, as they only
track the resources from individual namespaces, not the entire tree. To inspect hierarchical
quotas, look at the HRQ objects, or use the `kubectl hrq` command.
```